src/engine/Service.cpp

```diff
@@ -125,6 +125,31 @@ ProtoResult Service::computeResultImpl([[maybe_unused]] bool requestLaziness) {
   ad_utility::httpUtils::Url serviceUrl{
       asStringViewUnsafe(parsedServiceClause_.serviceIri_.getContent())};
 
+  // Receive updates about the RuntimeInformation from the service endpoint.
+  const std::string queryId = ad_utility::UuidGenerator()();
+  auto updateRuntimeInformation = [&]() {
+    try {
+      const std::string target = absl::StrCat("/watch/", queryId);
```

Comment:
This `watch` should be a global constant that is used both here and in the Websocket module.